app/src/components/base/Button/Button.tsx

```diff
@@ -1,41 +1,44 @@
 import React from 'react';
 import styled from '@emotion/styled';
-import { colors, pxToRem } from '../../../styled-theme';
+import { pxToRem } from '../../../styled-theme';
 
 export type ButtonType = 'submit' | 'reset' | 'button';
 
 export interface ButtonProps {
   readonly children: React.ReactNode;
-  readonly onClick?: () => void;
+  readonly onClick?: (params?: any) => void;
```

Comment:
This should be the proper typing

```suggestion
  readonly onClick?: (event: React.MouseEventHandler<HTMLButtonElement>) => void;
```